### PR TITLE
OK-9676: Optimized theme and components

### PR DESCRIPTION
**Pull request by franco-chan**
What Changed:
- Optimized the theme to make the UI cleaner
- Optimized the shadow token to make cards outstanding.
- Optimized `Header` `Sidebar` `BottomNavigation` to match the theme changed.
- Added BlurView to the Header component

### CODE REVIEW

**Review by revan-zhang — DISMISSED** [review]

**Review by originalix — DISMISSED** [review]

**Review by originalix — DISMISSED** [review]

File: packages/app/ios/Podfile.lock [hunk 1]
```diff
@@ -914,6 +920,10 @@ SPEC CHECKSUMS:
   Yoga: 99652481fcd320aefa4a7ef90095b95acd181952
   ZXingObjC: fdbb269f25dd2032da343e06f10224d62f537bdb
 
+<<<<<<< Updated upstream
```
Comment (ByteZhang1024) [inline packages/app/ios/Podfile.lock]: Git 忘记解决此处冲突？

File: packages/components/src/Layout/Header/index.tsx [hunk 2]
```diff
@@ -27,58 +28,72 @@ const Header: FC<HeaderProps> = ({ headerLeft, headerRight }) => {
...
+      pt={`${insets.top}px`}
+      alignItems="center"
+      justifyContent={isHorizontal ? 'flex-end' : 'space-between'}
+      borderBottomColor="divider" // TODO: change the color from transparent to divider while scrolling up
+      borderBottomWidth={StyleSheet.hairlineWidth}
+      px={2}
+      style={{
+        // @ts-expect-error
+        '-webkit-app-region': 'drag',
+        '-webkit-user-select': 'none',
```
Comment (ByteZhang1024) [inline packages/components/src/Layout/Header/index.tsx:46]: 应该是可以删除掉的，看是否符合预期（Desktop 使用鼠标拖动、双击最大化），这里估计是我忘记删除了:
'-webkit-app-region': 'drag',
'-webkit-user-select': 'none',

File: packages/app/ios/Podfile.lock [hunk 3]
```diff
@@ -914,6 +920,10 @@ SPEC CHECKSUMS:
...
 PODFILE CHECKSUM: a2bc322b38fb3c0adfe91ed01a15181b25e605a5
+=======
+PODFILE CHECKSUM: a27cd1d0dca9ce95aa5433dd4151fc79e6474ee1
+>>>>>>> Stashed changes
```
Comment (sunnylqm) [inline packages/app/ios/Podfile.lock]: merge conflict

File: packages/components/src/Layout/Header/index.tsx (hunk 2 above)
Comment (franco-chan) [reply packages/components/src/Layout/Header/index.tsx:46]: 我貌似没动这里。

**Review by revan-zhang — APPROVED** [review]

**Review by linleiqin — APPROVED** [review]

**Review by ByteZhang1024 — APPROVED** [review]

**Pull request merged**